corehq/apps/cloudcare/templates/formplayer/case_detail.html

```diff
@@ -14,6 +14,8 @@
   <td class="module-caselist-column"><img class="module-icon" src="<%= resolveUri(data) %>"/></td>
   <% } else if (style.displayFormat === 'Phone') { %>
   <td><a href="tel:<%- data %>"><%- data %></a></td>
+  <% } else if (markdown) { %>
+  <td><%= data %></td>
```

Comment:
Is it possible/worthwhile to name this something other than data, to convey that it has HTML in it? On the Python side, if we want to mark something as safe, we are going to have to put a comment next to it explicitly stating why it's safe, and I'd like something more explicit than just using markdown's interpolation over escaping here.

Reply:
+1 to adding a comment here.

Reply:
Do you have suggestions for https://github.com/dimagi/commcare-hq/pull/29557/files#diff-cd3ce94b57385fe87dc296bec57917470f84399b356c8b975c74b7f4d3fe1c01R179-R180

The reason I added the markdown flag there was that I wasn't sure how tight the coupling between this view and that controller were (e.g. another controller without the sanitization could be using this view). Would it be sufficient to rename markdown to sanitized to indicate that it's safe?

Reply:
To me, its more about `data` than it is about the markdown flag. When you transform it from raw data to markdown, it stops being data and becomes HTML. Seeing:
`<%= html %>` (or `markdownHTML`) leaves no doubt that you expect this to contain HTML tags that cannot be escaped.

Reply:
It's kinda tricky because `data` is used for the other cases as well like the other [styles](https://github.com/dimagi/commcare-hq/pull/29557/files/907c26d7569ec673a4d9ab89afad127ce5cd9ce1#diff-2d1246ef9efed6d5019cf222eff4ea055af84737d1f54e7a0de72b38268a7f18R16). I guess it probably makes sense to only sanitize it and store it in another variable `html` if it's not one of those styles.